src/server/routes/partner-details.route.test.js

```diff
@@ -666,7 +666,8 @@ describe("Partner Details Route: ", () => {
     });
 
     describe("When an error is thrown", () => {
```

Comment:
change describe text to make distinct from new test e.g. "When session.save throws an error"